Documentation/ApiOverview/PasswordPolicies/_PasswordCommand.php

```diff
@@ -0,0 +1,35 @@
+<?php
```

Comment:
This is not referenced anymore and can be removed?

Reply:
done